build_tools/cmake/iree_installed_test.cmake

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
```

Comment:
Nit:

```suggestion
# Copyright 2021 Google LLC
```

:-/

Reply:
Useless requirements drivel :(  I just got into the habit of typing 2020.

Reply:
The fact that anything needs to be typed there is a bug.

Reply:
Yeah I totally agree, but I talked to the OSS policy team and it appears that this really is a bug with the Apache License and IP law :-/ (IANAL, opinions are my own, and all that :stuck_out_tongue:)